### ISSUE TITLE
[BUG] stats variogram plotting

### ISSUE THREAD

**Issue by dbekaert** [dbekaert/RAiDER#198]
**Describe the bug**
Running the example provided with the PR
```
raiderStats.py --file UNRcombinedGPS_ztd.csv -w maps -b '36 40 -124 -119' -ti '2018-01-01 2019-12-31' --seasonalinterval '03-21 06-21' -variogramplot -variogram_per_timeslice --cpus all -verbose -grid_to_raster
```

Issue in create variograms:
```
(raider) MT-101939:Sim dbekaert$ raiderStats.py --file UNRcombinedGPS_ztd.csv -w maps -b '36 40 -124 -119' -ti '2018-01-01 2019-12-31' --seasonalinterval '03-21 06-21' -variogramplot -variogram_per_timeslice --cpus all -verbose -grid_to_raster
***Stats Function:***
***Variogram Analysis Function:***
Traceback (most recent call last):
  File "/Users/dbekaert/Software/python/miniconda37/envs/raider/bin/raiderStats.py", line 4, in <module>
    __import__('pkg_resources').run_script('RAiDER==0.0.1', 'raiderStats.py')
  File "/Users/dbekaert/Software/python/miniconda37/envs/raider/lib/python3.7/site-packages/pkg_resources/__init__.py", line 665, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/Users/dbekaert/Software/python/miniconda37/envs/raider/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1463, in run_script
    exec(code, namespace, namespace)
  File "/Users/dbekaert/Software/python/miniconda37/envs/raider/lib/python3.7/site-packages/RAiDER-0.0.1-py3.7-macosx-10.9-x86_64.egg/EGG-INFO/scripts/raiderStats.py", line 41, in <module>
    inps.variogram_per_timeslice
  File "/Users/dbekaert/Software/python/miniconda37/envs/raider/lib/python3.7/site-packages/RAiDER-0.0.1-py3.7-macosx-10.9-x86_64.egg/RAiDER/statsPlot.py", line 1086, in stats_analyses
    TOT_grids, TOT_res_robust_arr = make_variograms.create_variograms()
  File "/Users/dbekaert/Software/python/miniconda37/envs/raider/lib/python3.7/site-packages/RAiDER-0.0.1-py3.7-macosx-10.9-x86_64.egg/RAiDER/statsPlot.py", line 488, in create_variograms
    for i in sorted(list(set(self.df['gridnode']))):
TypeError: '<' not supported between instances of 'str' and 'int'
```

**Comment by sssangha** [comment]
@dbekaert I cannot replicate:
```
(RAiDER) [simran@centos raider_test]$ raiderStats.py --file products/UNRcombinedGPS_ztd.csv -w maps -b '36 40 -124 -119' -ti '2018-01-01 2019-12-31' --seasonalinterval '03-21 06-21' -variogramplot -variogram_per_timeslice --cpus all -verbose -grid_to_raster
***Stats Function:***
***Variogram Analysis Function:***
- Plot variogram range per gridcell.
- Plot variogram sill per gridcell.
```

Are you attempting this over the small test study I had specified?

As a sanity check, download the GNSS files as specified below and relaunch:
```
raiderDownloadGNSS.py --out products --years '2018,2019' --returntime '00:00:00' --bounding_box '36 40 -124 -119' --cpus all --verbose

raiderDownloadGNSS.py --out products --years '2018,2019' --returntime '00:00:00' --bounding_box '36 40 -124 -119' --cpus all --verbose
```

**Comment by dbekaert** [comment]
tried it on the server not on my laptop:

```
(RAiDER-gmao) [dbekaert@leffe weather_files]$ raiderStats.py --file products/UNRcombinedGPS_ztd.csv -w maps -b '36 40 -124 -119' -ti '2018-01-01 2019-12-31' --seasonalinterval '03-21 06-21' -variogramplot -variogram_per_timeslice --cpus all -verbose -grid_to_raster
***Stats Function:***
***Variogram Analysis Function:***
- Plot variogram range per gridcell.
/home/dbekaert/python/python36/envs/RAiDER-gmao/lib/python3.7/site-packages/cartopy/io/__init__.py:260: DownloadWarning: Downloading: https://naciscdn.org/naturalearth/50m/physical/ne_50m_land.zip
  warnings.warn('Downloading: {}'.format(url), DownloadWarning)
/home/dbekaert/python/python36/envs/RAiDER-gmao/lib/python3.7/site-packages/cartopy/io/__init__.py:260: DownloadWarning: Downloading: https://naciscdn.org/naturalearth/50m/physical/ne_50m_ocean.zip
  warnings.warn('Downloading: {}'.format(url), DownloadWarning)
/home/dbekaert/python/python36/envs/RAiDER-gmao/lib/python3.7/site-packages/cartopy/io/__init__.py:260: DownloadWarning: Downloading: https://naciscdn.org/naturalearth/10m/physical/ne_10m_coastline.zip
  warnings.warn('Downloading: {}'.format(url), DownloadWarning)
- Plot variogram sill per gridcell.
(RAiDER-gmao) [dbekaert@leffe weather_files]$ 
```

**Comment by sssangha** [comment]
@dbekaert so it works? And yeah, you usually get those warning messages on the first run when you attempt to download some prerequisite caropy maps.

Not sure I understand why it didn't work on your laptop though.

**Comment by dbekaert** [comment]
I pulled the 06 UTC file from the server and ran on that. Now testing with a local download file adn that works as well.
Can you test the 06 UTC file (global_products_06/UNRcombinedGPS_ztd.csv) we will be working from?

**Comment by dbekaert** [comment]
> @dbekaert so it works? And yeah, you usually get those warning messages on the first run when you attempt to download some prerequisite cartopy maps.

Might need some minor change as the logger is not catching on the formatting.

**Comment by jlmaurer** [comment]
Fixed by #200
